build.sbt

```diff
@@ -13,8 +13,8 @@ homepage := Some(url("https://github.com/codingwell/scala-guice"))
 useGpg := true
 
 libraryDependencies ++= Seq(
-  "com.google.inject" % "guice" % "4.1.0",
-  "com.google.inject.extensions" % "guice-multibindings" % "4.1.0",
+  "com.google.inject" % "guice" % "4.2.0",
+  "com.google.inject.extensions" % "guice-multibindings" % "4.2.0",

```

Comment:
https://github.com/google/guice/wiki/Guice42: `Multibindings are now in the Guice core artifact. The multibindings artifact is empty to prevent confusion during upgrading and will be removed in a later release.`

I think this should be removed

Reply:
Done